model/build.gradle

```diff
@@ -43,10 +43,11 @@ dependencies {
     api "fr.neolegal:exp4j:$exp4jVersion"
 
     dependencies {
-        testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
-        testImplementation 'org.mockito:mockito-core:5.18.0'
+        testImplementation 'org.junit.jupiter:junit-jupiter-api:5.12.2'
+        testImplementation 'org.mockito:mockito-core:5.12.0'
```

Comment:
Why is mockito downgraded?

Reply:
This one might have been a copy/paste mistake, testing with newer version.